### PR TITLE
VACMS-4695: Initial enable and config raven

### LINKED ISSUE #4695: Drupal Application integration with Sentry
## Acceptance Criteria
- [ ] Drupal Application integration with Sentry is configured using contrib [Raven](https://www.drupal.org/project/raven)
- [ ] PHP Errors, Exceptions, SQL Deadlocks from the Drupal application [DEV, STAGING, PROD] instance are flowing into Sentry

## implementation notes
- look into settings.php to differentiate between environments

### PR DESCRIPTION
**Pull request by indytechcook**
## Description

Closes #4695. 

## QA steps

As user _uid_ with _user_role_
1. Do this
1. Then that
1. Then validate Acceptance Criteria from issue
- [ ] This
- [ ] That
- [ ] The other thing

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by indytechcook** [comment]
Events being sent to sentry: 

image

**Comment by indytechcook** [comment]
To get tests to pass I have to `drush cim` then run tests.  I think this is because I added a new config split setup.  I had to do the same thing locally.

File: docroot/modules/custom/va_gov_build_trigger/src/Controller/FrontEndBranchAutocompleteController.php [hunk 1]
```diff
@@ -57,7 +57,7 @@ public static function create(ContainerInterface $container) {
         'department-of-veterans-affairs/content-build',
         'va_cms_bot_github_auth_token'
       ),
-      Git::get($container->get('va_gov.build_trigger.web_build_command_builder')->getPathToWebRoot()),
+      $container->get('va_gov.consumers.git.repository.factory')->getWebRepository(),
```
Comment (acrollet) [inline docroot/modules/custom/va_gov_build_trigger/src/Controller/FrontEndBranchAutocompleteController.php:60]: looks like `Drupal\va_gov_consumers\Git\Git` is no longer used in this class?

**Review by acrollet — COMMENTED** [review]
This makes sense to me and the config_split approach looks right. My only question is around how we configure the sentry DSN - will that be with an env variable? If so, do we need to wait for this code to be in before setting that up?

**Comment by indytechcook** [comment]
> This makes sense to me and the config_split approach looks right. My only question is around how we configure the sentry DSN - will that be with an env variable? If so, do we need to wait for this PR to be merged before setting that up?

duh, I had this entire thing planned out then just decided not to do it I guess.  I'm not sure what's going on in my mind.  It will be via env variables.  The sentry packages looks for `SENTRY_DSN`, `SENTRY_ENVIRONMENT`, and `SENTRY_RELEASE` to set the DSN.  This overrides anything passed into the class.

**Comment by indytechcook** [comment]
Devops PR: https://github.com/department-of-veterans-affairs/devops/pull/9661

**Comment by indytechcook** [comment]
PR for devops: https://github.com/department-of-veterans-affairs/devops/pull/9661

**Comment by acrollet** [comment]
> Devops PR: [department-of-veterans-affairs/devops#9661](https://github.com/department-of-veterans-affairs/devops/pull/9661)

looks good!

**Review by acrollet — APPROVED** [review]

**Pull request merged**